values-cs/strings.xml

```diff
@@ -2,126 +2,126 @@
-    <string name="how_to_play_blitz_paragraph1">Swipe left, right, up, or down to slide the tiles across the grid. When exactly <b>2 tiles</b> with the same number touch, they merge into one.</string>
-    <string name="how_to_play_blitz_paragraph2">New tiles are placed randomly on the grid at fixed intervals, <b>not</b> after each move. The game ends when the grid becomes full and no more tiles can be placed. Can you create the 2048 tile?</string>
-    <string name="how_to_play_fibonacci_paragraph1">Swipe left, right, up, or down to slide the tiles across the grid. When <b>2 consecutive Fibonacci numbers</b> touch, they merge into one to form the next Fibonacci number in the sequence.</string>
-    <string name="how_to_play_fibonacci_paragraph2">At the end of each move, <b>1 new tile</b> is placed randomly on the grid. The game ends when you can no longer make a move.</string>
-    <string name="how_to_play_fibonacci_hint"><b>Hint:</b> 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144&#8230;</string>
-    <string name="how_to_play_144_paragraph1">Swipe left, right, up, or down to slide the tiles across the grid. When <b>any number of tiles</b> with same number touch, they merge into one.</string>
-    <string name="how_to_play_144_paragraph2">At the end of each move, <b>1 new tile</b> is placed randomly on the grid. The game ends when you can no longer make a move. Can you create the 144 tile?</string>
-    <string name="how_to_play_144_hint"><b>Hint:</b> 2 &#215; 2 &#215; 2 &#215; 2 &#215; 3 &#215; 3 = 144</string>
-    <string name="tap_to_play_random_move">Tap the grid to play a random move!</string>
+    <string name="how_to_play_2048_paragraph1">Přejeď doleva, doprava, nahoru nebo dolů k posunutí dlaždice po ploše. 2 dlaždice se sloučí do jedné pokuď mají stejné čísla.</string>
+    <string name="how_to_play_2048_paragraph2">Po každém pohybu se přidá nová dlaždice kdekoliv na ploše. Hra končí pokuď již nelze provést tah. Dokážeš udělat dlaždici s číslem 2048?</string>
+    <string name="how_to_play_243_paragraph1">Přejeď doleva, doprava, nahoru nebo dolů k posunutí dlaždice po ploše. 3 dlaždice se sloučí do jedné pokuď mají stejné čísla.</string>
+    <string name="how_to_play_243_paragraph2">Po každém pohybu se přidá nová dlaždice kdekoliv na ploše. Hra končí pokuď již nelze provést tah. Dokážeš udělat dlaždici s číslem 243?</string>
+    <string name="how_to_play_16384_paragraph1">Přejeď doleva, doprava, nahoru nebo dolů k posunutí dlaždice po ploše. 2 nebo 4 dlaždice se sloučí do jedné pokuď mají stejné čísla.</string>
+    <string name="how_to_play_16384_paragraph2">Po každém pohybu se přídají 2 nové dlaždice kdekoliv na ploše. Hra končí pokuď již nelze provést tah. Dokážeš udělat dlaždici s číslem 16384?</string>
+    <string name="how_to_play_blitz_paragraph1">Přejeď doleva, doprava, nahoru nebo dolů k posunutí dlaždice po ploše. 2 dlaždice se sloučí do jedné pokuď mají stejné čísla.>/string>
```

Comment:
This line should end with "</string>". Right now it ends with ">/string>" which will result in a syntax error. :)